.pre-commit-config.yaml

```diff
@@ -60,14 +60,12 @@ repos:
         require_serial: true
         verbose: true
         additional_dependencies:
-          # the following are needed to run mypy successfully in the pre-commit virtualenv
-          - types-setuptools
+          # the following are needed to run mypy.
           - types-PyYAML
         args:
-          - --zero-exit
           - --output-format=text
         exclude: |-
           (?x)^(
-            tests/suppression/testdata/.*
-            |tests/tools/vulture/testpath/testfile\.py
```

Comment:
Linting test files would bring value, how many should be fixed before this work at the moment ?